### DISCUSSION TITLE
Markdown数学符号≠渲染错误

### DISCUSSION THREAD

**Discussion by Lay-Etamsylate** [F9y4ng/GreasyFork-Scripts#187] [问答专区(Question / Answer)] [answered]
![image](https://user-images.githubusercontent.com/71595854/226111239-c18981bb-3d08-4c27-9908-3c081900bf9a.png)
渲染错误为
![image](https://user-images.githubusercontent.com/71595854/226111269-4bfaf848-3cc0-4c27-be29-6bbd3ea5b3c9.png)
渲染字体设置为“苹方-简” 经测试，设置其他字体也会渲染错误
Markdown公式为 `$(A_i \neq B_i )$`

**Comment by F9y4ng** [comment, marked as answer]
反馈这类问题最好把访问地址带上。

如果你了解css，可以将公式部分的渲染排除，样例可参照#42 。
